crystals/parsers.py

```diff
@@ -692,6 +692,23 @@ def atoms(self) -> Iterable[Atom]:
             xs = tmpdata.get("_atom_site_fract_x")
             ys = tmpdata.get("_atom_site_fract_y")
             zs = tmpdata.get("_atom_site_fract_z")
+            Bs = tmpdata.get('_atom_site_B_iso_or_equiv') 
```

Comment:
Do these quantites not exist if the atomic coordinates are in cartesian coordinates (`cartesian=True`)?